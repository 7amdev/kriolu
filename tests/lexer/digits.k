12 912
12.90
89283
123.929
0.12129012 5.23 78.005
