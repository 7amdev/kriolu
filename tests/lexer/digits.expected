1 <number> '12' 
1 <number> '912'
2 <number> '12.90'
3 <number> '89283'
4 <number> '123.929'
5 <number> '0.12129012' 
5 <number> '5.23' 
5 <number> '78.005'